packages/perseus/src/widgets/__shared__/noop-validator.tsx

```diff
@@ -0,0 +1,20 @@
+import type {PerseusScore} from "../../types";
+
+/**
+ * Several widgets don't have "right"/"wrong" validation logic,
+ * so this just says to move on past those widgets
+ *
+ * TODO: widgets that use this probably shouldn't have any
+ * validation logic and the thing validating an exercise
+ * should just know to skip these
+ */
+function noopValidator(...args: any[]): PerseusScore {
```

Comment:
**nit**: accepting weakly typed `args` like this feels like it will just be confusing at the call sites. I think it'd be clearer to not accept any arguments.